### PR TITLE
added superseed testnet & Morph Testnet & RPC Worldchain fix

### PR DESCRIPTION
**Pull request by Fazleiw**
### Description

Add superseed testnet

### Backward compatibility

Yess

### Testing

Deployed using hyperlane CLI thru docker
Screenshot 2024-10-21 114810
relay message send tester
image
bartio to superseed 
image
explorer
image

### CODE REVIEW

**Review by Fazleiw — COMMENTED** [review]
added morph holesky too

explorer testing for morph
https://explorer-holesky.morphl2.io/tx/0x3ca8f5fe0f410c848cd856365361616cb9a506b24562616dc63774652f42b3b2
explorer hyperlane
https://explorer.hyperlane.xyz/message/0xea3e364aaf8e9afb4257b8a9777494e199283a7f322af1f835436694a9231496?data=eyJzdGF0dXMiOiJwZW5kaW5nIiwiaWQiOiI3NzUxNjk5IiwibXNnSWQiOiIweGVhM2UzNjRhYWY4ZTlhZmI0MjU3YjhhOTc3NzQ5NGUxOTkyODNhN2YzMjJhZjFmODM1NDM2Njk0YTkyMzE0OTYiLCJub25jZSI6MTYsInNlbmRlciI6IjB4YjNBRjhmQkM2M2FCQUVhMzFlMEE0ZERCMjY4ZTE1QTlhQjZkQzY3ZiIsInJlY2lwaWVudCI6IjB4NzQxMWRkYjBkZmQ3MGQ2MTA5ODI2ZTQzMDI4ZWZmMjc2YmM2MWZiYyIsIm9yaWdpbkNoYWluSWQiOjgwMDg0LCJvcmlnaW5Eb21haW5JZCI6ODAwODQsImRlc3RpbmF0aW9uQ2hhaW5JZCI6MjgxMCwiZGVzdGluYXRpb25Eb21haW5JZCI6MjgxMCwib3JpZ2luIjp7InRpbWVzdGFtcCI6MTcyOTQ5NTYxMzAwMCwiaGFzaCI6IjB4ZjQ3NzFmMjI2OWMwNWVmNDU1MzhjODgyNjE5MzBiZjY5MjZlNmI4NWVmNGRhNjA4YzM3NmRhYzdhNTVmN2QyMiIsImZyb20iOiIweGIzQUY4ZkJDNjNhQkFFYTMxZTBBNGREQjI2OGUxNUE5YUI2ZEM2N2YifSwiaXNQaU1zZyI6dHJ1ZX0=

File: chains/morphholeskytestnet/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  estimateBlockTime: 1
+  reorgPeriod: 1
+chainId: 2810
+deployer:
+  name: Fazleiw-Griffin
+  url: github.com/Fazleiw
+displayName: Morph Testnet
+domainId: 2810
+isTestnet: true
+name: morphholeskytestnet
```
[the remark this answers is not in the document]
Comment (Fazleiw) [reply chains/morphholeskytestnet/metadata.yaml:18]: suree no probs

File: chains/morphholeskytestnet/Logo.svg
Comment (paulbalaji) [inline chains/morphholeskytestnet/Logo.svg:1]: nit: rename to lowercase `logo.svg`

**Review by paulbalaji — COMMENTED** [review]
almost there, one more thing

**Pull request closed without merging**